apps/kyc-server/src/hooks/useKYCWebSocket.ts

```
@@ -0,0 +1,75 @@
+
+		return () => {
+			if (ws.readyState === WebSocket.OPEN) {
+				ws.close()
+			}
+		}
+	}, [userId, onUpdate])
+
+	return {
+		isConnected,
+		lastUpdate,
+		sendMessage: <T extends Record<string, any>>(message: T) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)